src/app/shared/form/form.service.ts

```diff
@@ -161,6 +161,15 @@ export class FormService {
       field.setErrors(error);
     }
 
+    // if the field in question is a group, pass down the error to its fields
+    if (field instanceof FormGroup && model instanceof DynamicFormGroupModel) {
```

Comment:
I'd restrict this approach to only concat group model (you can use `FormBuilderService.isConcatGroup`). Because a form group can be made of unrelated fields and in this way we are assigned the error to the sub-fields of a form group without distinction

Reply:
Done.